test/model/autocomplete_test.dart

```diff
@@ -812,20 +902,27 @@ void main() {
         RecentDmConversation(userIds: [1, 2], maxMessageId: 100),
       ]);
 
-      // Check the ranking of the full list of users.
+      // Check the ranking of the full list of mentions.
       // The order should be:
-      // 1. Users most recent in the current topic/stream.
-      // 2. Users most recent in the DM conversations.
-      // 3. Human vs. Bot users (human users come first).
-      // 4. Alphabetical order by name.
-      check(await getResults(topicNarrow, MentionAutocompleteQuery('')))
+      // 1. Wildcards.
```

Comment:
```suggestion
      // 1. Wildcards before individual users.
```

This numbered list isn't really a list of categories from first to last in the results, but more a list of ranking criteria from highest to lowest priority in their effect on the order of results. (E.g. compare the "human vs. bot users" item.)